### PR TITLE
Setup mobile skeleton with the first section

### PR DESCRIPTION
**Pull request by lucascomamala**
- Added HTML for the first section
- Added CSS for the first section
- Added nav bar (without functionality yet)
- Added background images for the first section
- Added linter files according to Microverse instructions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+# Portfolio Project
+
+GitHub repository for my personal portfolio
+
+## Built With
+
+- HTML
+- CSS
+- Figma template
+
```
Comment (wuletawwonte) [inline README.md:12]: - [ ] Your readme is missing important sections. Please add the missing sections from [Microverses readme template](https://github.com/microverseinc/readme-template). And please don't forget to customize its content to the details of your project.

File: README.md [hunk 2]
```diff
@@ -1 +1,34 @@
...
+ 
+Contributions, issues, and feature requests are welcome!
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- The Microverse Team
```
Comment (wuletawwonte) [inline README.md:34]: - [ ] [OPTIONAL] Your project is missing a license file. Please add an MIT license file in the root folder of your project. You can use [these template](https://github.com/microverseinc/readme-template/blob/master/MIT.md).

File: style.css [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+  line-height: 24px;
+  color: #fff;
+  margin: 12px 0.1px;
+  padding: 1px 24px;
+}
+
+.headline {
+  height: 590px;
+  background-color: #1c1a19;
+  background-image: url('./media/Illustration-header-mobile.svg');
```
Comment (wuletawwonte) [inline style.css:56]: - [ ] You did great adding the proper background for the section. But it is not visible in some case sensitive operating systems like linux. Please change the first character of the image file to `i`.

**Review by wuletawwonte — CHANGES_REQUESTED** [review]
# Hi @lucascomamala  👋🏿👋🏿👋🏿  

Good job so far!
There are some issues that you still need to work on to go to the next project! You have only a few more steps and you are done! 💪 💪 💪 

giphy

## TO HIGHLIGHT: 
- Your project is professional ✔️ 
- Correct Github Flow ✔️
- Your PR has a proper title and description ✔️
- Nice commit history ✔️
- Well structured files ✔️ 

## Required Changes ♻️
_Please check the comments under the review as well._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: style.css (hunk 3 above)
Comment (cwaku) [inline style.css:56]: [OPTIONAL] 
- [ ] Kindly have a look at the URL to the background image. You saved it in lowercase but referenced it here in uppercase. While that might work on some systems, others are case-sensitive and that prevents it from displaying for example on my system. 

image

**Review by cwaku — APPROVED** [review]
Hi @lucascomamala 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - [x] Good job overall following the template design 💯 
 - [x] Correct GitHub Flow 👍🏾 
 - [x] Good commit history 🥇 
 - [x] Good PR title/description 💯 
 - [x] All linters passed 🥇 
 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 3 above)
Comment (lucascomamala) [reply style.css:56]: @cwaku Thank you for catching that! The previous reviewer mentioned it but I only renamed the file and forgot to change it in the CSS

**Pull request merged**